relay-server/src/metrics_extraction/metrics_summary.rs

```diff
@@ -100,9 +100,9 @@ impl<'a> From<&'a SetValue> for MetricsSummaryBucketValue {
     fn from(set: &'a SetValue) -> Self {
         // For sets, we limit to counting the number of occurrences.
         MetricsSummaryBucketValue {
-            min: None,
-            max: None,
-            sum: None,
+            min: Some(0.into()),
+            max: Some(0.into()),
+            sum: Some(0.into()),
```

Comment:
Should we type these fields to be non-optional, if the consumer cannot handle them otherwise?

Reply:
According to the Kafka schema, any of these is required. So technically it should work if we just send `min: Some(0)` but keep the others `None` ... Unfortunately this is quite hard to express in the typesystem.